### ISSUE TITLE
Login isn't working + anon login request

### ISSUE THREAD

**Issue by nezvers** [the-mirror-gdp/the-mirror#18]
OS: Windows 10

Reproduction:
- launched windows version of Godot/Mirror editor
- imported suggested project from the `/mirror-godot-app` folder
- project opened with errors so restarted as suggested
- pressed F5 to launch project and was blocked by `Sign in` screen (No mention of it in the readme)
- tried to sign up (although that's already a sacrilegious act and questionable practice to see project actually working)
- after filling blanks, nothing happened. console had error print ```[Firebase Error] >> Encrypted Firebase Auth file does not exist```
- no mention of that in the Readme too

Debug log showed extra errors

```
E 0:00:00:0099   OpenXRAPI::load_supported_extensions: OpenXR: Failed to enumerate number of extension properties
  <C++ Error>    Condition "((result) < 0)" is true. Returning: false
  <C++ Source>   modules\openxr\openxr_api.cpp:154 @ OpenXRAPI::load_supported_extensions()
```
```
E 0:00:00:0115   Node::rpcp: Condition "!is_inside_tree()" is true. Returning: ERR_UNCONFIGURED
  <C++ Source>   scene\main\node.cpp:733 @ Node::rpcp()
```
```
E 0:00:00:0119   Node::_validate_child_name: Condition "!node_hrcr_count.ref()" is true.
  <C++ Source>   scene\main\node.cpp:1269 @ Node::_validate_child_name()
```
```
E 0:00:02:0696   ResourceFormatLoaderText::load: Cannot open file 'res://player/cameras/voxel_viewer/camera_voxel_viewer.tscn'.
  <C++ Error>    Condition "err != OK" is true. Returning: Ref<Resource>()
  <C++ Source>   scene\resources\resource_format_text.cpp:1642 @ ResourceFormatLoaderText::load()
```
```
E 0:00:02:0701   ResourceLoader::_load: Resource file not found: res://player/cameras/voxel_viewer/camera_voxel_viewer.tscn (expected type: PackedScene)
  <C++ Error>    Condition "!file_check->file_exists(p_path)" is true. Returning: Ref<Resource>()
  <C++ Source>   core\io\resource_loader.cpp:276 @ ResourceLoader::_load()
```
```
E 0:00:02:0710   ResourceLoaderText::_parse_ext_resource: res://player/cameras/camera_manager.tscn:35 - Parse Error: [ext_resource] referenced non-existent resource at: res://player/cameras/voxel_viewer/camera_voxel_viewer.tscn
  <C++ Source>   scene\resources\resource_format_text.cpp:163 @ ResourceLoaderText::_parse_ext_resource()
```
```
E 0:00:02:0736   ViewportTexture::get_height: Viewport Texture must be set to use it.
  <C++ Source>   scene\main\viewport.cpp:136 @ ViewportTexture::get_height()
```
```
E 0:00:02:0740   ViewportTexture::get_width: Viewport Texture must be set to use it.
  <C++ Source>   scene\main\viewport.cpp:126 @ ViewportTexture::get_width()
```
![image](https://github.com/the-mirror-gdp/the-mirror/assets/26794315/f2ef034f-9df3-4d50-a236-16854f188f0b)

**Comment by Kluskey** [comment]
Thanks for submitting; I believe the login issues are fixed, so I'll close this for now. Would you reopen it if the issue reappears?

The docs mention the login step here https://docs.themirror.space/docs/get-started

Logging in without an account is partially implemented but not finished, so the latter part would be a duplicate of https://github.com/the-mirror-gdp/the-mirror/issues/39

**Comment by Kluskey** [comment]
Also the docs mention the login step here https://docs.themirror.space/docs/get-started
